src/manual/tasks/workertypes.md

```diff
@@ -19,4 +19,7 @@ interchangeable and have identical configurations.
 The format of a task's `payload` property is specific to the worker that will
 execute it, so defining a task requires knowledge of worker type's
 configuration. If given a task with an inappropriate payload, a worker will
-resolve the task with the reason `malformed-payload`.
\ No newline at end of file
+resolve the task with the reason `malformed-payload`.
+
+You can explore the available worker types (organized by `provisionerId`) at
+https://tools.taskcluster.net/provisioners.
```

Comment:
`https://tools.taskcluster.net/provisioners/:provisionerId/worker-types`

Reply:
I want to make links, though, so the user can click and find what they are looking for without editing the URLs.  Maybe I can phrase that differently..